config/services.yaml

```diff
@@ -29,7 +29,7 @@ services:
         resource: '../src/Controller/'
         tags: ['controller.service_arguments']
 
-    App\CreationLog:
+    App\Controller\ExportController:
         arguments:
             $enableStats: '%env(bool:APP_ENABLE_STATS)%'
```

Comment:
I wonder if there's any point in being able to disable stats collection?  It seems like we could probably just drop this env var (could be a future patch of course, or even one before this one).

Reply:
Funny you ask; tools-db has gone down several times lately. Did you or someone disable this, or is there some other magic that makes it not error out? I don't remember seeing any errors. Anyway, that's what I figured this env var was for.

Reply:
FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI FUERDAI